packages/cli/src/lib/versioning/Lockfile.ts

```diff
@@ -79,7 +108,7 @@ export class Lockfile {
 
     for (const [key, value] of Object.entries(data)) {
       const [, name, range] = ENTRY_PATTERN.exec(key) ?? [];
-      if (!name) {
+      if (!name && key !== '__metadata') {
```

Comment:
Not sure if this is the he best way, or if there are more special keys. Will look into this a little more.